helper/staking/staking.go

```diff
@@ -134,12 +156,22 @@ func PredeployStakingSC(
 		return nil, fmt.Errorf("unable to generate DefaultStatkedBalance, %w", err)
 	}
 
+	//Validate min and max validators number
+	if err = validateMinMaxValidatorsNumber(params.MinValidatorCount, params.MaxValidatorCount); err != nil {
+		return nil, err
+	}
+
 	// Generate the empty account storage map
 	storageMap := make(map[types.Hash]types.Hash)
 	bigTrueValue := big.NewInt(1)
 	stakedAmount := big.NewInt(0)
+	minNumValidators := big.NewInt(int64(params.MinValidatorCount))
+	maxNumValidators := big.NewInt(int64(params.MaxValidatorCount))
```

Comment:
Please rename these values to `bigMinNumValidators` and `bigMaxNumValidators`, so it follows the naming convention from the rest of the method

Reply:
fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a